### DISCUSSION TITLE
Getting ETIMEDOUT error

### DISCUSSION THREAD

**Discussion by yubraj2022** [artilleryio/artillery#1442] [Artillery]
Hi
I have set config.http.timeout as 300 S but also i am getting etimeout error. here is the config setting and summary report.
Thanks much in advance. This group has been really helpful.

`
http: 
    timeout: 300
    extendedMetrics: true
  phases:
    - arrivalRate: 50
      duration: 30
      maxVusers: 60

VERSION INFO:

Artillery Core: 2.0.0-16
Artillery Pro:  3.0.0-9 

Node.js: v16.15.0       
OS:      win32

All VUs finished. Total time: 54 seconds

--------------------------------
Summary report @ 11:31:01(-0500)
--------------------------------

errors.ETIMEDOUT: .............................................................. 412
errors.Failed expectations for request https:// my-request..  1
errors.Failed expectations for request https:// my-request..  1
errors.Failed expectations for request https:// my-request..  1
errors.Failed expectations for request https:// my-request..  1
http.codes.202: ................................................................ 686
http.codes.500: ................................................................ 4
http.dns:
  min: ......................................................................... 0
  max: ......................................................................... 35
  median: ...................................................................... 0
  p95: ......................................................................... 4
  p99: ......................................................................... 7.9
http.request_rate: ............................................................. 16/sec
http.requests: ................................................................. 1102
http.response_time:
  min: ......................................................................... 391
  max: ......................................................................... 3726
  median: ...................................................................... 608
  p95: ......................................................................... 854.2
  p99: ......................................................................... 1408.4
http.responses: ................................................................ 690
http.tcp:
  min: ......................................................................... 282
  max: ......................................................................... 15370
  median: ...................................................................... 1300.1
  p95: ......................................................................... 15218.6
  p99: ......................................................................... 15218.6
http.tls:
  min: ......................................................................... 286
  max: ......................................................................... 3149
  median: ...................................................................... 295.9
  p95: ......................................................................... 354.3
  p99: ......................................................................... 699.4
http.total:
  min: ......................................................................... 998
  max: ......................................................................... 19207
  median: ...................................................................... 2101.1
  p95: ......................................................................... 16486.1
  p99: ......................................................................... 16819.2
plugins.expect.failed: ......................................................... 12
plugins.expect.failed.contentType: ............................................. 4
plugins.expect.failed.matchesRegexp: ........................................... 4
plugins.expect.failed.statusCode: .............................................. 4
plugins.expect.ok: ............................................................. 2058
plugins.expect.ok.contentType: ................................................. 686
plugins.expect.ok.matchesRegexp: ............................................... 686
plugins.expect.ok.statusCode: .................................................. 686
plugins.metrics-by-endpoint.my test api endpoint.codes.202: ..... 686
plugins.metrics-by-endpoint.response_time.my test api endpoint:
  min: ......................................................................... 391
  max: ......................................................................... 3726
  median: ...................................................................... 620.3
  p95: ......................................................................... 854.2
  p99: ......................................................................... 1408.4
vusers.completed: .............................................................. 686
vusers.created: ................................................................ 1102
vusers.created_by_name.my test api endpoint: ..................................... 1102
vusers.failed: ................................................................. 416
vusers.session_length:
  min: ......................................................................... 1000.7
  max: ......................................................................... 19209.3
  median: ...................................................................... 2101.1
  p95: ......................................................................... 16486.1
  p99: ......................................................................... 16819.2
vusers.skipped: ................................................................ 398`

**Comment by hassy** [comment]
hi @yubraj2022 👋 do you have access to logs or monitoring for the backend services you're testing to see what's happening? `ETIMEDOUT` is reported when the backend service does not respond within the timeout window. It is very high in your configuration, but could still be happening.
